web/concrete/controllers/panel/detail/page/composer.php

```diff
@@ -128,7 +129,9 @@ protected function save() {
 
 			/// set the target
 			$configuredTarget = $pagetype->getPageTypePublishTargetObject();
-			$targetPageID = $configuredTarget->getPageTypePublishTargetConfiguredTargetParentPageID();
+			if ($configuredTarget instanceof PageTypePublishTargetConfiguration) {
+				$targetPageID = $configuredTarget->getPageTypePublishTargetConfiguredTargetParentPageID();
+			}
 			if (!$targetPageID) {
```

Comment:
what about adding this?
```php
} else {
    $targetPageID = null;
}
```
That way, we avoid php warnings about trying to access undefined variables...